.config/nextest.toml

```diff
@@ -33,3 +33,10 @@ default-filter = "package(integration-tests)"
 path = "junit.xml"
 store-success-output = true
 store-failure-output = true
+
+# Per-test overrides for libvirt tests that create VMs
+# These tests race on creating libvirt base disks, so they need to run serially
+# VMs get 4 vcpus by default, so require 4 threads to prevent any parallelism
```

Comment:
OK this definitely improves our CI reliability...but it's still quite slow. I'll chase that separately.